flood/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:15.04
```

Comment:
Is this the same as the other ones?  I'm just asking b/c it's otherwise a full download the first time you use it (vs there other images).